src/fdb5/tools/fdb-hammer.cc

```diff
@@ -1188,9 +1269,11 @@ void FDBHammer::executeWrite() {
     if (config_.execution.itt) {
         eckit::Timer barrier_timer;
         barrier_timer.start();
-        //        Barrier.block();
-        barrier(config_.parallel.ppn, config_.parallel.nodelist, config_.parallel.barrierPort,
-                config_.parallel.barrierMaxWait);
+        Barrier barrier{(size_t)config_.parallel.ppn, config_.parallel.nodelist, config_.parallel.barrierPort,
```

Comment:
This code should be:

Barrier(config_).barrier();

Not sure that the init/fini functions add any value.